src/Lorentz/Contracts/BaseDAO.hs

```diff
@@ -38,7 +38,7 @@ baseDaoContract = defaultContract $ contractName "FA2 smart contract" $ do
 fa2Handler :: Entrypoint FA2.Parameter Storage
 fa2Handler =
   entryCase @FA2.Parameter (Proxy @PlainEntrypointsKind)
-    ( #cTransfer /-> transfer
+    ( #cTransfer /-> transfer defaultPermissionsDescriptor
```

Comment:
It appears the as per the spec, the contract implements the FA2 specified default permissions description, so at least that aspect does not have to be configurarble, so we don't have to do check the permissions descriptor values while doing permission checks during transfer. So we probably don't need this code https://github.com/tqtezos/baseDAO/pull/22/files#diff-454ccfe639f4a10802990f520295d821b3d444efa51dac4690b092cbdb1cd158R74-R99, and the default behavior can be just hardcoded.

Reply:
Okay, but we still have to do this checks, don't we? 
```
  checkAdmin OwnerOrOperatorTransfer = do
      dup
      Lorentz.sender
      if IsEq
      then nop
      else do
        dupTop2
        pair
        Lorentz.sender
        pair
        dig @3
        getField #sOperators
        dig @2
        mem
        if_ (dug @2) (failUsing [mt|#fA2_NOT_OPERATOR|])
```

Reply:
About this 
> I think we should also implement permissions_descriptor entrypoint as well here

I think, that since it is not part of FA2 it should be left for another issue or something.

Reply:
> Okay, but we still have to do this checks, don't we?

Yes. 

> I think, that since it is not part of FA2 it should be left for another issue or something.

You are right. 

The entrypoint is part of FA2,

https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-12/tzip-12.md#exposing-permissions-descriptor

but as the spec say, FA2 allows us to omit the entrypoint if the contract implements default permissions. And we implement default permissions here so we don't need to include the entrypoint here. 

So I suppose you can remove the `permissionsDescriptor` function.

Reply:
How to fail in Lorentz, with a custom error, that requires some params like  `failCustom_ #fA2_INSUFFICIENT_BALANCE` with `("required" :! Natural, "present" :! Natural)` arguments